test/contains.js

```diff
@@ -28,6 +28,36 @@ describe('contains', function() {
     assert.strictEqual(R.contains(new Just([42]), [new Just([42])]), true);
   });
 
+  it('dispatches to `indexOf` method', function() {
```

Comment:
I've updated this pull request to dispatch to `contains` if applicable. Note also the updated docstring.